README.adoc

```diff
@@ -40,7 +40,7 @@ and for Gradle:
 $ docker build --build-arg JAR_FILE=build/libs/*.jar -t myorg/myapp .
 ```
 
-Of course, once you have chosen a build system, you don't need the `ARG` - you can just hard code the jar location. E.g. for Maven:
+Of course, once you have chosen a build system, you don't need the `ARG` -- you can just hard code the jar location. E.g. for Maven:
```

Comment:
We use two hyphens here so that Asciidoctor will create an em dash for us. So this is not a change we would want to make.